doc/containers.md

```diff
@@ -235,6 +235,33 @@ lxc config device add <container> <device-name> nic nictype=sriov parent=<sriov-
 To tell LXD to use a specific unused VF add the `host_name` property and pass
 it the name of the enabled VF.
 
+### Type: infiniband
+LXD supports two different kind of network types for infiniband devices:
+
+ - `physical`: Straight physical device passthrough from the host. The targeted device will vanish from the host and appear in the container.
+ - `sriov`: Passes a virtual function of an SR-IOV enabled physical network device into the container.
+
+Different network interface types have different additional properties, the current list is:
+
+Key                     | Type      | Default           | Required  | Used by         | API extension | Description
+:--                     | :--       | :--               | :--       | :--             | :--           | :--
+nictype                 | string    | -                 | yes       | all             | infiniband    | The device type, one of "physical", or "sriov"
+name                    | string    | kernel assigned   | no        | all             | infiniband    | The name of the interface inside the container
+hwaddr                  | string    | randomly assigned | no        | all             | infiniband    | The MAC address of the new interface
+mtu                     | integer   | parent MTU        | no        | all             | infiniband    | The MTU of the new interface
+parent                  | string    | -                 | yes       | physical, sriov | infiniband    | The name of the host device or bridge
+
+To create a `sriov` `infiniband` device use:
+
+```
+lxc config device add <container> <device-name> infiniband nictype=sriov parent=<sriov-enabled-device>
+```
+
+To create a `physical` `infiniband` device use:
+
+```
+lxc config device add <container> <device-name> infiniband nictype=physical parent=<device>
```

Comment:
I'd flip the two examples as physical is the one which requires the less setup and so is most likely to work.

Reply:
addressed